include/InputHandling/ThumbstickHandler.hpp

```diff
@@ -6,10 +6,17 @@
 #include "UnityEngine/XR/XRNode.hpp"
 #include "Enums.hpp"
 
-DECLARE_CLASS_CUSTOM(TrickSaber::InputHandling, ThumbstickHandler, TrickSaber::InputHandling::InputHandler,
-    DECLARE_PRIVATE_FIELD(StringW, _inputString);
-    DECLARE_INSTANCE_METHOD(float, GetInputValue);
+#include "UnityEngine/Object.hpp"
 
-    DECLARE_CTOR(ctor, UnityEngine::XR::XRNode node, TrickSaber::ThumbstickDir thumbstickDir, float threshold, bool isReversed);
+namespace TrickSaber::InputHandling {
+    struct ThumbstickHandler : public TrickSaber::InputHandling::InputHandler {
 
-)
\ No newline at end of file
+        SafePtr<Il2CppString> _inputString;
```

Comment:
Mentioned in discord as well, but could this be made to be a `ConstString&` where the ctor assigns the reference in the field initializers. The referenced ConstStrings could then live in the global space or as static fields on this type